Doc/#457 - Clarify the explanation of the `validity_period` attribute (#460)
Avaiga/taipy-doc: docs/manuals/core/entities/data-node-mgt.md

- _**edits**_: The ordered list of `Edit^`s, representing the successive modifications of the data node.
- _**version**_: The string indicates the application version of the data node to instantiate.
    If not provided, the current version is used.

> - _**validity_period**_: The duration since the last edit date which the data node can be considered up-to-date.

Reviewer: which -> for which

>     Once the validity period is passed, the data node is considered stale and relevant tasks will run

Reviewer: is passed -> has passed

>     even if they are skippable (see the [Task management page](task-mgt.md) for more detail).

Reviewer: more details

>     even if they are skippable (see the [Task management page](task-mgt.md) for more detail).

Reviewer 2: neat !

>     If _validity_period_ is set to `None`, the data node is always up-to-date.

Reviewer: Please use stars instead of underscores for italics. Just like you did above.
Please do NOT use the code style (backtick) for Python values (None, True, False...)

- _**edits**_: The ordered list of `Edit^`s, representing the successive modifications of the data node.
- _**version**_: The string indicates the application version of the data node to instantiate.
    If not provided, the current version is used.
- _**validity_period**_: The duration since the last edit date which the data node can be considered up-to-date.

>     Once the validity period is passed, the data node is stale and relevant tasks will not be skippable.

Reviewer 2: ```suggestion
    Once the validity period is passed, the data node is considered stale and relevant tasks will run even if they are skippable.
```

Reviewer 2: You can also add a reference to the entity/task-mgt part of the doc that explain the skippable.
